cockpit/src/index.html

```diff
@@ -26,13 +26,12 @@
 
     <link rel="stylesheet" href="../base1/patternfly.css">
     <link href="subscriptions.css" rel="stylesheet">
-
-    <script type="text/javascript" src="../base1/cockpit.js"></script>
-    <script type="text/javascript" src="../*/po.js"></script>
-    <script type="text/javascript" src="index.js"></script>
 </head>
 
 <body>
     <div id="app"></div>
+    <script type="text/javascript" src="../base1/cockpit.js"></script>
+    <script type="text/javascript" src="../*/po.js"></script>
+    <script type="text/javascript" src="index.js"></script>
```

Comment:
Is this (adding all `<script></script>` at the end of `<body>`) recommended approach? Isn't better to use `async` loading of javascript files?

Reply:
In this case, it should not be significantly different. We do not have content to display prior to the loading of the bundle, so there's no real point IMO. Might be worth investigation in the future into using `async` with `<script>` if we get reports that the UI takes a while to load, but I think it'd require some re-architecture...